content/blog/2023/nextflow-with-gbatch.md

```diff
@@ -0,0 +1,291 @@
+
+### Running an RNAseq pipeline with Google Cloud Batch
+
+Welcome to our RNAseq tutorial using Nextflow and Google Cloud Batch! RNAseq is a powerful technique for studying gene expression and is widely used in a variety of fields, including genomics, transcriptomics, and epigenomics. In this tutorial, we will show you how to use Nextflow, a popular workflow management tool, to perform an RNAseq analysis on Google Cloud Batch, a scalable cloud-based computing platform.
+
+Nextflow allows you to easily develop, execute, and scale complex pipelines on any infrastructure, including the cloud. Google Cloud Batch enables you to run batch workloads on Google Cloud Platform (GCP), with the ability to scale up or down as needed. Together, Nextflow and Google Cloud Batch provide a powerful and flexible solution for RNAseq analysis.
+
+We will walk you through the entire process, from setting up your Google Cloud account and installing Nextflow to running an RNAseq pipeline and interpreting the results. By the end of this tutorial, you will have a solid understanding of how to use Nextflow and Google Cloud Batch for RNAseq analysis. So let's get started!
+
+### Setting up Google Cloud CLI (gcloud)
+
+In this tutorial, you will learn how to use the gcloud command-line interface to interact with the Google Cloud Platform and set up your /"_
+Google Cloud account for use with Nextflow. If you do not already have gcloud installed, you can follow the instructions found [here](https://cloud.google.com/sdk/docs/install) to install it. Once you have gcloud installed, run the command gcloud init to initialize the CLI. You will be prompted to choose an existing project to work on or create a new one. For the purpose of this tutorial, we will create a new project. Name your project "my-rnaseq-pipeline". There may be a lot of information displayed on the screen after running this command, but you can ignore it for now.
+
+### Setting up Batch and Storage in Google Cloud Platform
+
+#### Enable Google Batch
+
+According to the [official Google documentation](https://cloud.google.com/batch/docs/get-started), "Batch is a fully managed service that lets you schedule, queue, and execute [batch processing](https://en.wikipedia.org/wiki/Batch_processing) workloads on Compute Engine virtual machine (VM) instances. Batch provisions resources and manages capacity on your behalf, allowing your batch workloads to run at scale."
+
+If you're using macOS, the first step is to download the `beta` command group. You can do this by executing:
+
+```bash
+$ gcloud components install beta
+```
+
+Then, enable billing for this project. You will first need to get your account id with
+
+```bash
+$ gcloud beta billing accounts list
+```
```

Comment:
Nitpicking point, but I think you're supposed to use `console` instead of `bash` if you include the `$`.

Reply:
I have followed what other blog posts around here do 😬 Not sure what's the correct way. Does it really matter? 👀

Reply:
No, not really